checkpoint/ddb/retryer_test.go

```diff
@@ -0,0 +1,32 @@
+package ddb
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func TestRetyer(t *testing.T) {
```

Comment:
What do you think if we update the test case name into "TestDefaultRetryer" instead?